placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesAutocompleteTextView.java

```diff
@@ -171,18 +170,20 @@ public void onItemClick(final AdapterView<?> parent, final View view, final int
                 }
             }
         });
-        if(clearEnabled)
-            enableClearButton(true);
+        if(clearEnabled) {
+            enableClearButton(context, true);
+        }
         super.setDropDownBackgroundResource(R.drawable.pacv_popup_background_white);
     }
 
-    private void enableClearButton(boolean value){
+    private void enableClearButton(Context context, boolean value){
         if(!value) {
             this.setCompoundDrawables(null, null, null, null);
             return;
         }
-        if(imgClearButton == null)
-            imgClearButton = getResources().getDrawable(R.drawable.ic_clear_black_24dp);
+        if(imgClearButton == null) {
+            imgClearButton = ContextCompat.getDrawable(context, R.drawable.ic_clear_black_24dp);
```

Comment:
I think this needs to be `AppCompatResources` for proper vector support, also we'll need to add this to the `build.gradle`:

```
    android.defaultConfig.vectorDrawables.useSupportLibrary = true
```

Reply:
Yea of course I mix them up a bit